simulation_ws/.rosinstall

```diff
@@ -1 +1,2 @@
 - git: {local-name: src/deps/xacro, uri: "https://github.com/AAlon/xacro", version: ros2-find}
```

Comment:
So if we're using pre-built urdf models now, is this `xacro` dependency still required?

Reply:
Looks like it's not. Have updated https://github.com/aws-robotics/turtlebot3-description-reduced-mesh/pull/3 removing it from import there and then we can remove it from this .rosinstall.